### PR TITLE
Internal/deploy asa vault

### PR DESCRIPTION
**Pull request by Jesulonimi21**
This PR uses the updated xbacked-contracts sdk and contains the test ids for testing a fake dAlgo vault

### CODE REVIEW

File: examples/asa-vault/VaultOwner.js [hunk 1]
```diff
@@ -9,9 +9,9 @@ dotenv.config();
   VAULTS.TestNet.algo.vaultId;
   console.log(VAULT_ID);
   const STABLECOIN = process.env.STABLE_COIN;
+  const COLLATERALTOKEN = process.env.COLLATERAL_TOKEN;
```
Comment (AwolDes) [inline examples/asa-vault/VaultOwner.js:12]:
```suggestion
  const COLLATERAL_TOKEN = process.env.COLLATERAL_TOKEN;
```
Just a stylistic preference

File: src/utils.ts [hunk 2]
```diff
@@ -17,6 +17,12 @@ export const VAULTS = {
       vaultId: 152242747,
       liquidatorDiscount: 0.045,
     },
+    dAlgo:{
```
Comment (AwolDes) [inline src/utils.ts:20]:
```suggestion
    gAlgo:{
```

**Review by AwolDes — CHANGES_REQUESTED** [review]
Looking good here, minor suggestions

File: src/utils.ts (hunk 2 above)
Comment (Jesulonimi21) [reply src/utils.ts]: @AwolDes 
Done, please let me know if you think its good to go to create a new SDK version.

File: examples/getAllAccounts.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ dotenv.config();
       // indexer key
       token,
       // indexer url
-      'https://algoindexer.testnet.algoexplorerapi.io',
+      'https://mainnet-api.algonode.network',
```
Comment (JCastrejonE) [inline examples/getAllAccounts.js:21]: Is algoexplorer failing to do the boxes requests on these examples too?

File: .gitignore [hunk 4]
```diff
@@ -108,7 +108,7 @@ dist
 **/docs
 
 # Lib Folder
-/lib
+# /lib
```
Comment (JCastrejonE) [inline .gitignore:111]: What's the actual reason we want to stop using npm deployments and use the git files directly?

**Review by JCastrejonE — CHANGES_REQUESTED** [review]
Approving but there's a small change required for the examples

**Review by JCastrejonE — APPROVED** [review]

File: examples/getAllAccounts.js (hunk 3 above)
Comment (Jesulonimi21) [reply examples/getAllAccounts.js:21]: I hven't tried algoexplorer yet

File: package.json [hunk 5]
```diff
@@ -63,7 +63,7 @@
     "@randlabs/myalgo-connect": "^1.1.1",
     "@reach-sh/stdlib": "0.1.13-rc.0",
     "@types/jsdom": "^16.2.13",
-    "@xbacked-dao/xbacked-contracts": "0.0.27",
+    "@xbacked-dao/xbacked-contracts": "1.0.2",
```
[the remark this answers is not in the document]
Comment (Jesulonimi21) [reply package.json:66]: ok, great

**Pull request merged**